README.md

```diff
@@ -57,7 +57,31 @@ Prior to using Helm to deploy applications to the OpenShift cluster, the CAS tea
 
 ## Terraform in CAS repos
 
-See an example of our containerized Terraform process in an OpenShift job that is integrated into a the 'cas-registration' Helm chart. It deploys at the pre-install, pre-upgrade hooks. The Terraform scripts are located in the `/terraform` subdirectory in the chart, which is then pulled in via a ConfigMap utilized by the job at `/templates/backend/job/terraform-apply.yaml`.
+### Components
+
+#### `~/helm/terraform-job/`
+
+This repo contains a Helm chart that contains a job that will import and run Terraform files. It deploys at the pre-install, pre-upgrade hooks. This chart references secrets and config that is deployed to a namespace when a project is provisioned by *`cas-pipeline`* (credentials, project_id, kubeconfig, terraform backendconfig).
+
+#### `~/terraform/modules/provision-storage-and-accounts`
+
+In tandem with the Helm chart is a Terraform module that creates GCP storage buckets, service accounts to access those buckets (admins and viewers) and injects those credentials into OpenShift for usage. These modules are pulled in via a configMap which pulls all files from the *parent chart's* `/terraform` directory.
+
+### Usage
+
+1. Import the Helm Chart into your project's main chart as a dependency.
+2. Add a subdirectory to the project chart's directory called `/terraform`. This is where you will place your `main.tf` and `variables.tf` files (and any other Terraform modules that you want imported and applied)
+3. Add a template into your chart with `{{ include "terraform-job.moduleImportMap" . }}`. This will use the partial from the terraform-job chart to pull in the terraform files from the project chart's context.
```

Comment:
We won't need these steps for v1, the main and variables files will be bundled.

Reply:
Rewrote some docs in 2286add to reflect the bundling.